sdk/rotation/schedule.go

```diff
@@ -13,7 +13,7 @@ import (
 const (
 	// Minimum allowed value for rotation_window
 	minRotationWindowSeconds = 3600
-	parseOptions             = cron.Second | cron.Minute | cron.Hour | cron.Dom | cron.Month | cron.Dow
+	parseOptions             = cron.Minute | cron.Hour | cron.Dom | cron.Month | cron.Dow
```

Comment:
In production, schedules using seconds should be disallowed. Only useful in testing. Matches behavior with the DB Engine as well